app/assets/stylesheets/components/_card.scss

```diff
@@ -34,51 +34,35 @@
   }
 }
 
+.show {
```

Comment:
Maybe confusing : I think "show" in css is more appropriate for an animation, like "active" or "hidden".

It could be a class for Javascript functions